model/registry/user.yaml

```diff
@@ -0,0 +1,59 @@
+groups:
+  - id: registry.user
+    prefix: user
+    type: attribute_group
+    brief: "Describes information about the user that is relevant to the event."
+    attributes:
+      - id: domain
+        type: string
+        stability: experimental
+        brief: >
+          Name of the directory the user is a member of. For example, an LDAP or Active Directory domain name.
+        examples: ['internal.example.com']
+      - id: email
+        type: string
+        stability: experimental
+        brief: >
+          User email address.
+        examples: ['a.einstein@example.com']
+      - id: full_name
+        type: string
+        stability: experimental
+        brief: >
+          User's full name
+        examples: ['Albert Einstein']
+      - id: hash
+        type: string
+        stability: experimental
+        brief: >
+          Unique user hash to correlate information for a user in anonymized form.
+        note: >
+          Useful if `user.id` or `user.name` contain confidential information and cannot be used.
+        examples: ['364fc68eaf4c8acec74a4e52d7d1feaa']
+      - id: id
+        type: string
+        stability: experimental
+        brief: >
+          Unique identifier of the user.
+        examples: ['S-1-5-21-202424912787-2692429404-2351956786-1000']
+      - id: name
```

Comment:
maybe `alias`? it'd be aligned with the brief "Short name or login/username of the user." better

Reply:
Hm, I think `name` is default field name for the user, even self explanatory. It is also very common and widely used. for example [OCSF](https://schema.ocsf.io/1.2.0/objects/user?extensions=) is also using just `name` for it